src/js/video-js-plugin/MqttHandler.js

```diff
@@ -125,27 +127,27 @@ export default class MqttHandler extends Component {
       this.player.on('changesrc', this.onChangeSource);
     });
 
-    this.updateIOV(iov);
+    this.updateIov(iov);
 
     // this.iov.on('unsupportedMimeCodec', (error) => {
```

Comment:
i don't recall, but since this is in the part of the code that deals with videojs, I am not comfortable removing it.  i added an @todo - also, in looking into this, it appears that this event was moved from the iov to the iovPlayer, so I made the necessary changes to declare the event in the iovPlayer